backoffice/backoffice-domain-service/ballerina/Dependencies.toml

```diff
@@ -232,6 +232,9 @@ dependencies = [
 	{org = "ballerina", name = "jballerina.java"},
 	{org = "ballerina", name = "lang.int"}
 ]
+modules = [
```

Comment:
add these same changes to dependencies.template.toml file as well

Reply:
Added.